core/block_render_svg_vertical.js

```diff
@@ -974,6 +976,55 @@ Blockly.BlockSvg.prototype.renderDraw_ = function(iconWidth, inputRows) {
 };
 
 /**
+ * Give the block an attribute 'data-shapes' that lists its shape[s]
+ * @private
+ */
+Blockly.BlockSvg.prototype.renderClassify_ = function() {
+  var shapes = [];
+  
+  if(this.outputConnection) {
+    if(this.isShadow_) {
+      shapes.push("input");
+    } else {
+      shapes.push("reporter");
+    }
+    if(this.edgeShape_ === Blockly.OUTPUT_SHAPE_HEXAGONAL) {
+      shapes.push("boolean");
+    } else if(this.edgeShape_ === Blockly.OUTPUT_SHAPE_ROUND) {
+      shapes.push("string");
```

Comment:
It looks like this also apples to color pickers, dropdowns that use a shadow block, etc. "round" is clearly the best option here